### DISCUSSION TITLE
Transferring Data from SMT Apps to Fossify Apps

### DISCUSSION THREAD

**Discussion by RCMoonpie** [FossifyOrg/General-Discussion#226] [Q&A]
I came here with the question about how to transfer data from SMT apps on my current phone to newly installed Fossify apps on a new phone, but it was not found as a top level discussion.  After searching, I found that it was asked by someone else in a comment under another discussion that has an unrelated title.

Thankfully, `Aga-C` answered it.

So (hopefully) to help out any future visitors with the same question, I'm posting this at top level. Please let me know if I've overstepped my bounds.

Aga's answer:
> Some of the SMT apps have data export in Settings. Then in Fossify apps you can import it, also in Settings.

_Originally posted by @Aga-C in https://github.com/FossifyOrg/General-Discussion/discussions/167#discussioncomment-10069174_

Dziękuję, @Aga-C!

**Comment by naveensingh** [comment]
I guess we can close this, everyone would have migrated by now :)
